app/Http/Controllers/Entity/PostMoveController.php

```diff
@@ -59,12 +53,20 @@ public function move(MovePostRequest $request, Entity $entity, EntityNote $entit
     {
 
         $newEntity = Entity::where(['id' => $request['entity']])->first();
+        $this->authorize('update', $newEntity->child);
         try {
             $newNote = $this->service
                 ->moveEntityNote($entityNote, $request->only('entity', 'copy'));
+            if (isset($request['copy'])) {
+                return redirect()
+                ->route($newEntity->pluralType() . '.show', [$newEntity->child->id, '#post-' . $newNote->id])
+                ->with('success', __('entities/notes.move.copy_success', ['name' => $newNote->name,
```

Comment:
Lots of copy-paste here when all we need is to change the success messsage. This could be refactored to be
```
$success = 'move_success';
if ($request->has('copy')) {
    $success = 'copy_success';
}
return redirect()...
```